packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -76,6 +76,9 @@ class ImagePicker {
   /// The `source` argument controls where the image comes from. This can
   /// be either [ImageSource.camera] or [ImageSource.gallery].
   ///
+  /// Where iOS supports HEIC images, Android 8 and below doesn't. Android 9 and above only support HEIC images if used
```

Comment:
nit: I'm not sure about the grammar here, maybe:

```
HEIC images are supported on iOS and Android versions greater than 9.
On Android 9 and above, HEIC images can only be picked when the `maxWidth` and `maxHeight` parameters are set.
```

(assuming I got the but on size modification correctly :) which I'm not sure of)